pages/magic.tsx

```diff
@@ -0,0 +1,47 @@
+import { createSessionCookie } from '@lib/helpers/backend/session-cookie';
```

Comment:
I think it would be nice to add a short documentation note describing what this endpoint is for. I initially thought that this was an alternative UI for generating magic links as opposed to the page where magic links go to.